### PR TITLE
clear state if the network changes

### PR DESCRIPTION
**Pull request by futurepaul**

### CODE REVIEW

**Comment by futurepaul** [comment]
closes #176

File: frontend/src/components/SettingStringsEditor.tsx [hunk 1]
```diff
@@ -12,6 +12,13 @@ export default function SettingStringsEditor() {
 
     async function handleSaveSettings() {
         try {
+            let existingNetwork = getExistingSettings().network;
+            if (existingNetwork !== nodeManagerSettings.network) {
+                if (window.confirm("Changing networks will delete your node's state. This can't be undone!")) {
+                    localStorage.clear();
+                    window.location.reload();
```
Comment (benthecarman) [inline frontend/src/components/SettingStringsEditor.tsx:19]: Do we need to write to storage the network on confirm?

Comment (TonyGiorgio) [reply frontend/src/components/SettingStringsEditor.tsx:19]: Yeah I think after localStorage.clear() it needs to write network.

**Review by TonyGiorgio — CHANGES_REQUESTED** [review]

File: frontend/src/components/SettingStringsEditor.tsx (hunk 1 above)
Comment (futurepaul) [reply frontend/src/components/SettingStringsEditor.tsx:19]: yeah this happens in the `setup` function

**Pull request merged**